hera_cal/io.py

```diff
@@ -1007,7 +1007,7 @@ def write_cal(fname, gains, freqs, times, flags=None, quality=None, total_qual=N
     '''
 
     # get antenna info
-    ant_array = np.array(sorted(map(lambda k: k[0], gains.keys())), np.int)
+    ant_array = np.unique(map(lambda k: k[0], gains.keys())).astype(np.int)
```

Comment:
`np.unique([int(k[0]) for k in gains])` is much more readable and succinct in my mind, but it's a matter of taste.